test/framework/vsphere.go

```diff
@@ -167,6 +172,15 @@ func WithBottleRocket121() VSphereOpt {
 	}
 }
 
+func WithPrivateNetwork() VSphereOpt {
```

Comment:
Any reason we want to separate the env vars here vs reusing the network env var for the private use case? Is it because we only define each of the env vars once and we need a way to separate the configuration?